File: Mahasiswa.py
# Definisi kelas Mahasiswa
class Mahasiswa:
    # Konstruktor kelas untuk inisialisasi objek
    def __init__(self, nama, nim, jurusan):
        self.nama = nama
        self.nim = nim
        self.jurusan = jurusan

    # Metode untuk menampilkan aktivitas belajar
    def kegiatan(self):
        print(f"{self.nama} sedang silat silat ditengah jalan.")
    # Metode untuk menampilkan aktivitas belajar
    def kegiatan2(self):
        print(f"{self.nama} sedang midioin orang silat tengah jalan.")

# Implementasi penggunaan kelas Mahasiswa
mahasiswa1 = Mahasiswa("Yanto", "123456", "Informatika")
mahasiswa2 = Mahasiswa("Syafira", "654321", "Sistem Informasi")

# Menggunakan metode objek
mahasiswa1.kegiatan()  # Output: Budi sedang belajar.
mahasiswa2.kegiatan2()  # Output: Ani sedang belajar.

File: Mahasiswa.java
//Implementasi penggunaan kelas mahasiswa
public class Main {
// Definisi kelas Mahasiswa
public static class Mahasiswa {
    // Atribut kelas
    private String nama;
    private String nim;
    private String jurusan;

    // Konstruktor kelas untuk inisialisasi objek
    public Mahasiswa(String nama, String nim, String jurusan) {
        this.nama = nama;
        this.nim = nim;
        this.jurusan = jurusan;
    }

    // Metode untuk mendapatkan nama mahasiswa
    public String getNama() {
        return nama;
    }

    // Metode untuk mengubah nama mahasiswa
    public void setNama(String nama) {
        this.nama = nama;
    }
        // Metode untuk mendapatkan nim mahasiswa
    public String getNim() {
        return nim;
    }

    // Metode untuk mengubah nim mahasiswa
    public void setNim(String nim) {
        this.nim = nim;
    }
        // Metode untuk mendapatkan jurusan mahasiswa
    public String getJurusan() {
        return jurusan;
    }

    // Metode untuk mengubah jurusan mahasiswa
    public void setJurusan(String jurusan) {
        this.jurusan = jurusan;
    }

    // Metode untuk menampilkan aktivitas belajar
    public void kegiatan() {
        System.out.println(nama + " dari jurusan " + jurusan +" dengan nim " + nim + " sedang melawan orang tua." );
    }
    public void kegiatan2() {
        System.out.println(nama + " dari jurusan " + jurusan +" dengan nim " + nim + " sedang adu silat dengan orang tua." );
    }
}
    public static void main(String[] args) {
        // Membuat objek Mahasiswa
        Mahasiswa mahasiswa1 = new Mahasiswa("Mas Jambrong", "321321", "Informatika");
        Mahasiswa mahasiswa2 = new Mahasiswa("Desi", "123123", "Sistem Informasi");

        // Menggunakan metode objek
        mahasiswa1.kegiatan();  // Output: Budi sedang belajar.
        mahasiswa2.kegiatan2();  // Output: Ani sedang belajar.
    }
}
